package/doc/sphinx/source/documentation_pages/selections.rst

```diff
@@ -104,6 +99,22 @@ moltype *molecule-type*
     select by molecule type, e.g. ``moltype Protein_A``. At the moment, only
     the TPR format defines the molecule type.
 
+Pattern matching
+-------
+
+The pattern matching notation described bellow is used to specify patterns for matching strings:
+
+``?`` 
+    Is a pattern that will match any single character. For example,
+    ``resname T?R`` selects residues named "TYR" and "THR".
+``*`` 
+    Is a pattern that will match multiple characters.  For example,
+    ``GL*`` selects all strings that start with "GL" such as "GLU",
+    "GLY", "GLX29", "GLN".
+``[seq]``
```

Comment:
I believe that fnmatch will also support [!seq] , I assume this would also work here.

Either way, you should try to introduce a test for [seq] (and [!seq] if it works).